package.json

```diff
@@ -20,5 +20,8 @@
   "bugs": {
     "url": "https://github.com/JustinHsie/eats/issues"
   },
-  "homepage": "https://github.com/JustinHsie/eats#readme"
+  "homepage": "https://github.com/JustinHsie/eats#readme",
+  "dependencies": {
+    "uuidv4": "^6.2.6"
```

Comment:
We don't need this here I think, you are using uuid and only for client/package.json